config/crds/tenancy.kcp.dev_clusterworkspacetypes.yaml

```diff
@@ -44,6 +54,37 @@ spec:
                 description: additionalWorkspaceLabels are a set of labels that will
                   be added to a ClusterWorkspace on creation.
                 type: object
+              allowedChildWorkspaceTypes:
+                default: []
+                description: "allowedChildWorkspaceTypes is a list of ClusterWorkspaceTypes
+                  that can be created in a workspace of this type. \n By default,
+                  no type is allowed. This means no other workspace can be nested
+                  within a workspace of the given type. The name `*` allows any child
+                  type to be nested."
+                items:
+                  description: ClusterWorkspaceTypeName is a name of a ClusterWorkspaceType
+                  pattern: ^[A-Z][a-zA-Z0-9]+$
```

Comment:
must be fully qualified.

Reply:
Let's follow up with straw-men hierarchies and implement as needed